README.md

```diff
@@ -1,2 +1,40 @@
 # renovate-config
+
 A sharable config preset for renovate used by SMG automotive.
+
+## Setup
+
+Enable Renovate in your repository and create a `renovate.json` file in the root of your project:
+
+````json
+{
+  "extends": [
+    "github>smg-automotive/renovate-config"
+  ]
+}
+````
+
+By default, renovate will pin all **dev** dependencies and uses SemVer ranges for the others. If you work on a project
+that
+is **not** `required()` by any other package, add `github>smg-automotive/renovate-config:web` to renovate.json.
```

Comment:
might be worth linking to the documentation as the reasoning is quite complex in practice...

https://docs.renovatebot.com/dependency-pinning/